app/src/test/kotlin/com/intive/picover/parties/viewmodel/PartiesViewModelTest.kt

```diff
@@ -37,7 +36,9 @@ class PartiesViewModelTest : ShouldSpec(
 
 		val dispatcher = UnconfinedTestDispatcher()
 		val partiesRemote: List<PartyRemote> = mockk()
-		val parties: List<Party> = mockk()
+		val parties: List<Party> = mockk {
+			every { isEmpty() } returns false
+		}
```

Comment:
Try to avoid mocking behavior of classes from std lib.
Instead of mockking `isEmpty`, let's proceed with the "real" list object and use mockk as a list item
`val parties: List<Party> = listOf(mockk())`

Reply:
Done